server/jetstream_helpers_test.go

```diff
@@ -924,6 +924,31 @@ func (c *cluster) createLeafNodesWithTemplateAndStartPort(template, clusterName
 	return lc
 }
 
+// Helper function to close and disable leafnodes.
+func (s *Server) closeAndDisableLeafnodes() {
+	var leafs []*client
+	s.mu.Lock()
+	for _, ln := range s.leafs {
+		leafs = append(leafs, ln)
+	}
+	// Disable leafnodes for now.
+	s.leafNodeEnabled = false
+	s.mu.Unlock()
+
+	for _, ln := range leafs {
+		ln.closeConnection(Revocation)
+	}
+}
+
+// Helper to set the remote migrate feature.
+func (s *Server) setJetStreamMigrateOnRemoteLeaf() {
```

Comment:
This is so that you don't have to modify the test's template config to have the new config param, correct?

Reply:
Correct.